assets/js/components/consent-mode/__snapshots__/ConsentModeSetupCTAWidget.test.js.snap

```diff
@@ -35,9 +35,12 @@ exports[`ConsentModeSetupCTAWidget should render the widget 1`] = `
                     class="googlesitekit-setup-cta-banner__description"
                   >
                     <p>
-                      Consent mode interacts with your Consent Management Platform (CMP) or custom implementation for obtaining visitor consent, such as a cookie consent banner. 
+                      <span>
+                        Consent mode interacts with your Consent Management Platform (CMP) or custom implementation for obtaining visitor consent, such as a cookie consent banner.
+                      </span>
+                       
                       <a
-                        aria-label="Learn more about consent mode (opens in a new tab)"
```

Comment:
Part of the aria-label is lost here. Is this expected?

Reply:
I did think it wasn't worth passing more props here again and to just use the default standard label that Link generates. The extra "about consent mode" doesn't really add much value here. But I've added this now as it might help in the future too for more custom aria labels.